pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
   "click ~= 8.0",
   "click-option-group ~= 0.5",
   "gitpython ~= 3.0",
+  "httpx ~=0.27",
   "requests ~= 2.25",
   "jinja2 ~= 3.1",
   "python-gitlab ~= 4.0",
```

Comment:
If we were to make a change, we should instead prevent the broken release of `python-gitlab` from being selected. Utilizing the `! 4.11.0` syntax.

With this line 28 addition of httpx, is not required.

This is the recommended manner of fixing a broken dependency library.